pkg/asset/agent/installconfig.go

```diff
@@ -78,6 +78,9 @@ func (a *OptionalInstallConfig) validateInstallConfig(ctx context.Context, insta
 	if err := a.validateSupportedPlatforms(installConfig); err != nil {
 		allErrs = append(allErrs, err...)
 	}
+	if err := a.validatePlatformsByName(installConfig); err != nil {
```

Comment:
This semms unnecessary as `a.validateSupportedPlatforms` internally calls the same function.

Reply:
Thanks for spotting it! Looks like a typo